src/Uno.UI/Behaviors/VisibleBoundsPadding.cs

```diff
@@ -144,8 +145,13 @@ internal VisibleBoundsDetails(FrameworkElement owner)
 
 				_visibleBoundsChanged = (s2, e2) => UpdatePadding();
 
-				owner.LayoutUpdated += (s, e) => UpdatePadding();
-				owner.Loaded += (s, e) => ApplicationView.GetForCurrentView().VisibleBoundsChanged += _visibleBoundsChanged;
+				owner.SizeChanged += (s, e) => UpdatePadding();
```

Comment:
Unfortunately `SizeChanged` won't work here. The VisibleBoundsPadding depends on the view's absolute position in the window, which may be affected by changes anywhere in the view's hierarchy, without its own size necessarily changing. We absolutely need to hook onto `LayoutUpdated` here.

Reply:
Ok...  if we add both it works...

@davidjohnoliver

Thanks for you greate reviews, as always.